### ISSUE TITLE
Getting network problem on my local PC ?

### ISSUE THREAD

**Issue by fy1128** [lrsjng/h5ai#459]
I am just trying to upgrade from 0.25.2 to 0.27, and after i replace the old files with the new one's, my chrome and firefox become hard to load the webpage, just loading and without nothing shown all the time. 

I think maybe there is some network related problem in my area, i am in china. But it works fine on my VPS located on Los Angeles of course.

Now i try to find some message debugged with chrome developer tools, and i get some images here:

![debug with chrome 3](https://cloud.githubusercontent.com/assets/358214/8208936/c6d6a49e-153c-11e5-864e-e98eb11a1ee4.png)
![debug with chrome 1](https://cloud.githubusercontent.com/assets/358214/8208935/c6d4e0d2-153c-11e5-8114-d9eb58ec9c7b.png)
![debug with chrome 2](https://cloud.githubusercontent.com/assets/358214/8208937/c6d7d8e6-153c-11e5-96d0-46947609bb88.png)

**Comment by UltimateByte** [comment]
Hey,
I think your issue might be the same as mine. I made a fresh install of the last version for the first time and my pages have pain to load. (See my issue opened just before yours).

It could be related to server configuration. What OS are you using on both of your servers ? What HTTP server ? The difference could come from here.

**Comment by fy1128** [comment]
@UltimateByte My local server is running with xampp on windows 7, the HTTP server is apache the same with my ubuntu VPS. i don't know how to make to work with my own PC now, i got nothing in the error logs from apche and php...

**Comment by UltimateByte** [comment]
So it works OK with your Ubuntu VPS and Apache but not your Windows Xampp (Apache) server ?
I read that it had issues on windows... I guess it has many issues in the end, and the developer may be on holidays. It probably needs to be fixed btw, i'm considering finding another script like this... Sadly, it seems to be the best out there... If only it was working :(

**Comment by h404bi** [comment]
Works fine in my xampp stack, did you upgrade to 0.27.0 by overlaying old version's files? If yes, maybe you should try to have a fresh install.

**Comment by UltimateByte** [comment]
It was a first and so fresh install for me. Got 5 other websites + some other file explorer for FastDL of SRCDS (Valve games) servers : They all work great.

**Comment by fy1128** [comment]
@h404bi No, i had deleted the old one completely. I had also tested it on my other windows PC, got the same problem...

The page just gets loading all the time, never stop...

**Comment by UltimateByte** [comment]
Do you have many files and folders ? Some files or folders bigger than 1GB ? I read somewhere that it could be the issue with the script not managing it correctly.

**Comment by h404bi** [comment]
Or you may try [dev builds](http://release.larsjung.de/h5ai/develop/), though I think it should be the server config problem, not h5ai.

**Comment by UltimateByte** [comment]
Well i tried latest build, folders still take forever to load.
Just as i'm saying here https://github.com/lrsjng/h5ai/issues/458
As all my other websites are fast like hell, the issue might be the h5ai not supporting the latest apache2 (or php or whatever it is) version, don't you think ?

**Comment by fy1128** [comment]
@UltimateByte I just test it with my opensuse Tumbleweed (apache 2.4.12 and php5.6.*), it works fine.

@h404bi  and the last dev build doesn't work on my windows 7 with xampp either.

I just tried to do a fresh installation with the last xampp + php5.6.8 on windows, but stuck again. I think it is some thing wrong with windows or xampp, i am not sure...

**Comment by UltimateByte** [comment]
You have to admitt it's curious to have such issues with a fresh install or a perfectly working server for several wordpress and so...
I personally have Apache 2.4.10-10 and php 5.6.9 (latest for Debian)

**Comment by lrsjng** [comment]
Hi all (@fy1128, @UltimateByte, @h404bi),

there is a known problem regarding file sizes on win-servers. Please try to change this [line](https://github.com/lrsjng/h5ai/blob/v0.27.0/src/_h5ai/server/php/inc/class-util.php#L131):

~~~php
   if (PHP_INT_SIZE < 8) {
~~~

into that
~~~php
   if (false) {
~~~

In the latest dev build you can find this line [here](https://github.com/lrsjng/h5ai/blob/master/src/_h5ai/private/php/core/class-util.php#L107).


to disable a size calculation workaround on servers running a 32bit PHP version (all windows servers). Maybe that solves it (at the cost of wrong filesizes on files > 2GB).

Other things to try: disable all extensions. Some extensions are very CPU intensive in case of large file/folder collections. Enable them one by one or in groups to check what works best for you.

**Comment by fy1128** [comment]
@lrsjng Thanks, but still got the same result with the lastest release (0.27) and the lastes dev build.

Could you tell me how to disable the extensions? h5ai's or the php's? I had changed all the enabled's value from true to false in the options.json.

**Comment by UltimateByte** [comment]
Maybe he's talking about options.json in conf folder ?
